metrics/IdentityManagement/PasswordLength/PasswordLength.yml

```diff
@@ -0,0 +1,13 @@
+# ====== Metadata ======
+id: PasswordLength
+category: IdentityManagement
+description: This rule assesses whether a [Resource] that offers the property [SecurityFeature.Authenticity.PasswordBasedAuthentication], has [p1:length] set correctly.
+implementationGuidelines:
+  AMOE: "How long should passwords be?"
+version: "1.0"
+comments: This metric evaluates the length of passwords. It ensures that the length is set to a sensible minimum value, which is 5 characters as default.
```

Comment:
It's better not to include the target value in the comments; instead, write in more general.